image/docker-compose.yml

```diff
@@ -110,10 +110,7 @@ services:
         max-file: "3"
 
   laboratory:
-    image: kinecosystem/laboratory
-    build:
-      dockerfile: Dockerfile.laboratory
-      context: .
+    image: kinecosystem/laboratory:1
```

Comment:
why is this tagged `:1`? laboratory doesn't have version 1 and it's misleading to use the version tag for that. just use  `:latest`

Reply:
don't forget to remove the dockerhub image tagged `:1`